project/tests/base_test.py

```diff
@@ -13,7 +17,50 @@ def spceff(self, name="spce.xml"):
         return foyer.Forcefield(forcefield_files=str(abs_path) + "/" + name)
 
     @pytest.fixture
-    def trappe_ua(
-        self,
-    ):
+    def job_gsdfile(self, tmp_job):
+        filename = tmp_job.fn("trajectory.gsd")
+        create_gsd(filename)
+        return tmp_job
+
+    @pytest.fixture
+    def tmp_project(self):
+        with signac.TemporaryProject(name="test") as p:
+            return p
+
+    @pytest.fixture
+    def tmp_job(self, tmp_project, statepoint={"a": 0}):
+        tmp_project.open_job(statepoint).init()
+        for job in tmp_project:
+            return job
+
```

Comment:
This needs the `@pytest.fixture` decorator

Reply:
This is for the `trappe_ua` fixture. It did not select multiple lines. Whoops.

Reply:
yeah, that's my bad! I've fixed it in a later commit.